src/test/java/hudson/plugins/sonar/utils/SonarUtilsTest.java

```diff
@@ -45,34 +41,6 @@
   @Rule
   public TemporaryFolder temp = new TemporaryFolder();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "TemporaryFolder". ([squid:S2924](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2924))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.jenkins-ci.plugins%3Asonar&pullRequest=90&issues=AWIv-I78_wH-bYHl3c0G&open=AWIv-I78_wH-bYHl3c0G)

[ISSUE_KEY]: AWIv-I78_wH-bYHl3c0G